src/main/java/StationConnection.java

```diff
@@ -0,0 +1,81 @@
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.Reader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class StationConnection
+{
+    private Gson gson = new Gson();
+
+    public List<Integer> getConnections(int stationId) throws IOException
+    {
+        //splitting the string station Json gives into the different lines
+        Reader readerStation = Files.newBufferedReader(Paths.get("src/main/resources/Subwaystations.json"));
```

Comment:
Be aware, you are reading this file in multiple times. This will be unacceptable for next week's homework.